internlm/solver/optimizer/hybrid_zero_optim.py

```diff
@@ -502,6 +508,20 @@ def step(self, closure=None):
 
         return self._step(closure=closure)
 
+    def _get_norm_with_moe_layers(self, norm_groups):
```

Comment:
When calculating the norm for model with MoE, 
- Should we first perform `all_reduce` on gradients from different expert groups, and then compute the norm? 
- Or should we first calculate the norm for each expert group, and then `all_reduce` the norms of different expert groups?